### PR TITLE
It seems vet on windows is unnecessary

### PR DESCRIPTION
**Pull request by lunny**
Fix the CI randomly failed.

### CODE REVIEW

**Comment by jolheiser** [comment]
I would argue that vet on Windows is still useful. Consider when someone creates new OS-specific code and may forget to add copyright. 

Aside from `gitea-vet` specifically, it would probably be nice to run the Go vet against it as well.

I haven't had a chance to take a look at what failed, but perhaps we could instead fix that.

**Review by 6543 — APPROVED** [review]

File: .drone.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ steps:
     pull: always
     image: golang:1.15
     commands:
-      - make golangci-lint vet
+      - make golangci-lint
```
Comment (6543) [inline .drone.yml]:
```suggestion
      - make golangci-lint
      - go vet ./....
```

Or split make target "vet" ...

**Review by 6543 — CHANGES_REQUESTED** [review]

File: .drone.yml (hunk 1 above)
Comment (lunny) [reply .drone.yml]: It's unnecessary, we have did that on before step.

**Comment by lunny** [comment]
@jolheiser But in fact, the drone run on the container `go1.15`, it's a Linux container. We can compile gitea-vet.exe, but we annot run it.

**Comment by jolheiser** [comment]
> @jolheiser But in fact, the drone run on the container `go1.15`, it's a Linux container. We can compile `gitea-vet.exe`, but cannot run it.

Ah, yep you are right. I still think we should probably vet the Windows code (and other tags, like PAM) to make sure that any changes to them are vetted properly.

In this scenario, I guess we really just need to be able to run the linux `gitea-vet` binary against the code with Windows tags.

File: .drone.yml (hunk 1 above)
Comment (6543) [reply .drone.yml]: but this should -go vet- the windows code too ...

**Comment by silverwind** [comment]
There has to be some better solution than having to run all lint/vet multiple times based on TAGS or GOOS because that method does not scale with the number of build tags. Can't we just somehow tell these tools to check all files, regardless of tags?

File: .drone.yml (hunk 1 above)
Comment (lunny) [reply .drone.yml]: It's unnecessary. We have done that on non windows lint.

**Comment by lunny** [comment]
@silverwind It depends on the tools. But as I know, golangci-lint doesn't support scan all the files.

**Review by 6543 — APPROVED** [review]

**Comment by silverwind** [comment]
Opened https://github.com/golangci/golangci-lint/issues/1646. Maybe it'll give us some clarification or lead to a feature to lint all files. I don't really know how these linters work so it may as well just not be possible.

**Comment by lunny** [comment]
@jolheiser @silverwind @6543 I have changed the PR, could you review it again?

**Review by 6543 — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**